tools/tests/test_docker_container_fs_changes_table.py

```diff
@@ -0,0 +1,43 @@
+#!/usr/bin/env python3
+
+#  Copyright (c) 2014-present, Facebook, Inc.
+#  All rights reserved.
+#
+#  This source code is licensed in accordance with the terms specified in
+#  the LICENSE file found in the root directory of this source tree.
+
+import unittest
+import docker
+
+import test_base
+
+docker_exceptions = (docker.errors.ContainerError, docker.errors.ImageNotFound, docker.errors.APIError)
+
+class FsChangesTableTest(unittest.TestCase):
+    def setUp(self):
+        self.binary = test_base.getLatestOsqueryBinary('osqueryi')
+        self.osqueryi = test_base.OsqueryWrapper(command=self.binary)
+        self.client = None
+        self.container = None
+        try:
+            self.client = docker.from_env()
+            self.container = self.client.containers.run("ubuntu:18.04", command="tail -f /dev/null", detach=True);
+            self.container.exec_run("touch xxx")
+        except docker_exceptions as e:
+            print(f"Failed in creating test container: {e}") 
+
+    def tearDown(self):
+        if self.container:
+            self.container.stop()
+        if self.client:
+            self.client.close()
+    
+    def testFsChangesTable(self):
```

Comment:
If someone runs the test and they do not have `docker` installed then the test will print the error message from `setUp` but then expect that `self.container` is usable. This mean we require `docker` for tests to run.

Reply:
I am not sure if this is a reasonable expectation for people developing osquery, should we expect developers to have `docker` installed?

Reply:
Makes sense, will fix it, thanks!

Reply:
Done